src/ansys/dpf/core/results.py

```diff
@@ -263,7 +263,12 @@ def eval(self):
         >>> fc = disp.on_all_time_freqs.eval()
 
         """
-        fc = self.__call__().outputs.fields_container()
+        outputs = self.__call__().outputs
+        if hasattr(outputs, "fields_container"):
+            fc = outputs.fields_container()
+        else:
+            # Support operators where the first output is not named "field_container"
+            fc = outputs[0]()
```

Comment:
@PProfizi Would it be a very breaking change to make this the default? I mean, is the `if` part necessary, doesn't this also handle fields_container?

Reply:
hey @rafacanton , osrry I just updated the description of the PR, you were too fast :)

Reply:
actually @rafacanton  you're right, the logic in the else should also handle the first case... thanks for noticing that!